lua/cmp/config/mapping.lua

```diff
@@ -62,7 +62,7 @@ mapping.preset.cmdline = function(override)
         if cmp.visible() then
           cmp.select_prev_item()
         else
-          feedkeys.call(keymap.t('<C-z>'), 'n')
+          feedkeys.call(keymap.t('<S-Tab>'), 'n')
```

Comment:
feedkeys is problematic as sometimes people change the default. `normal!` is the way.

Reply:
@eyalk11 do you have an example for how would that be used here?